bundles/AdminBundle/Security/User/User.php

```diff
@@ -15,161 +15,9 @@
 
 namespace Pimcore\Bundle\AdminBundle\Security\User;
 
-use Pimcore\Model\User as PimcoreUser;
-use Scheb\TwoFactorBundle\Model\Google\TwoFactorInterface as GoogleTwoFactorInterface;
-use Symfony\Component\Security\Core\User\EquatableInterface;
-use Symfony\Component\Security\Core\User\UserInterface;
-
 /**
- * Proxy user to pimcore model and expose roles as ROLE_* array. If we can safely change the roles on the user model
- * this proxy can be removed and the UserInterface can directly be implemented on the model.
+ * @deprecated and will be removed in Pimcore 11. Use \Pimcore\Security\User\User instead.
  */
-class User implements UserInterface, EquatableInterface, GoogleTwoFactorInterface
+class User extends \Pimcore\Security\User\User
```

Comment:
Should we do the deprecation of classes like this: 
https://github.com/pimcore/pimcore/blob/6.9/models/DataObject/ClassDefinition/Data/Multihref.php 

Then we can issue a proper deprecation message as well.

Reply:
agree, I would do this `User` class and rest are services, which are deprecated already https://github.com/pimcore/pimcore/blob/583a156af99429addbef734f49510aac03cdbc22/bundles/AdminBundle/Resources/config/security_services.yaml#L107